### PR TITLE
Change dialog

### PR DESCRIPTION
**Pull request by nielslyngsoe**
fixes https://github.com/umbraco/Umbraco-CMS/issues/9484

see issue for test notes

### CODE REVIEW

**Comment by warrenbuckley** [comment]
Code looks fine to me

File: src/Umbraco.Web.UI/Umbraco/config/lang/en.xml [hunk 1]
```diff
@@ -2531,9 +2531,10 @@ To manage your website, simply open the Umbraco back office and start adding con
     <key alias="endTitle">End preview mode</key>
     <key alias="openWebsiteLabel">Preview website</key>
     <key alias="openWebsiteTitle">Open website in preview mode</key>
-    <key alias="returnToPreviewHeadline">Preview content?</key>
-    <key alias="returnToPreviewDescription">You have ended preview mode, do you want to continue previewing this content?</key>
-    <key alias="returnToPreviewButton">Preview</key>
+    <key alias="returnToPreviewHeadline">Preview website?</key>
+    <key alias="returnToPreviewDescription">ou have ended preview mode, do you want to enable it again to view the latest saved version of your website?</key>
```
Comment (rbottema) [inline src/Umbraco.Web.UI/Umbraco/config/lang/en.xml:2535]: Small typo here (and on en_us.xml)

Comment (warrenbuckley) [reply src/Umbraco.Web.UI/Umbraco/config/lang/en.xml:2535]: Glad someone got their glasses on. Thanks for the spot @rbottema

Comment (nielslyngsoe) [reply src/Umbraco.Web.UI/Umbraco/config/lang/en.xml:2535]: @rbottema thanks a lot, good spotting, I have made a fix for it :-)

**Pull request merged**